Library/Homebrew/download_strategy.rb

```diff
@@ -389,6 +389,23 @@ def _fetch
   end
 end
 
+# Will convert "https://downloads.sourceforge.net/project/..." URLs to mirrorservice.org
+# In case environment variable HOMEBREW_NO_INSECURE_REDIRECT is set, so the Sourceforge is not ultimately broken
+class CurlSourceForgeSecureMirrorDownloadStrategy < CurlDownloadStrategy
+  def fetch
+    unless ENV["HOMEBREW_NO_INSECURE_REDIRECT"].nil?
```

Comment:
Well is this change the only thing holding this PR from being merged? Also one more question, should I squash commits using rebase, so I don't spam the Homebrew git tree a lot. Thanks